src/components/KantoSVG.html

```diff
@@ -544,4 +544,14 @@
         , databind: "click:function(){MapHelper.openShipModal()}, css:{ unlockedTown: player.highestRegion() >= 1}"
         })
      %>
+
+     <%- include('templates/mapOverlay',
+         { name: "SS-Anne"
+         , x: 50, y: 35
+         , width: 8, height: 3
+         , databind: "click:function(){MapHelper.openShipModal()}, css:{ unlockedTown: player.highestRegion() >= 1}"
+         , visible:  "player.highestRegion() >= 0"
```

Comment:
You say that this only appears once you reach Johto, did you mean it only *unlocks* or was this meant to be a 1 here? I can see it as soon as I unlock the map on a new save.

Reply:
😅 Yeah, supposed to be a `1` forgot to change it after testing
```suggestion
         , visible:  "player.highestRegion() >= 1"
```